src/solver/ts-generator/xcast/xcast.cpp

```diff
@@ -85,21 +89,18 @@ void XCast::exportTimeSeriesToTheOutput(Progression::Task& progression, Predicat
         String filename;
```

Comment:
I don't know why the current function **exportTimeSeriesToTheOutput**(...)  in a member of class **XCast** : it does not use a data / function member of  **XCast** (apart from **study**, ..., and now **pWriter**, ok).
Maybe we should make it an external function and so make **XCast** thinner. Ok, not a priority.